src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestAdsAbcdProductsPriority extends TemplateNoFirstLoad {
+
+  private static final String WIKIA = "project43";
+  private static final String URL_ARTICLE = "/SyntheticTests/ProductsPriority/OutstreamOverABCD";
+  private static final String URL_ARTICLE_WITH_DEBUG_MODE = URL_ARTICLE + "?appnexusast_debug_mode";
+
+  private static final String ABCD_LINE_ITEM_ID = "4417483196";
+  private static final String APPNEXUS_LINE_ITEM_ID = "417473960";
+
+
+  @Test(
+      groups = "AbcdProductPriorityOasis"
+  )
+  public void adsAppNexusVideoAdIsDisplayedIfAbcdAdIsTargetedOnThisSameArticleOasis() {
+    AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(WIKIA, URL_ARTICLE_WITH_DEBUG_MODE));
+    ads.scrollToSlot(AdsContent.INCONTENT_PLAYER);
+    verifyAppNexussAdIsDisplayed(ads);
+
+    ads.scrollToSlot(AdsContent.TOP_LB);
+    verifyAbcdAdIsNotDisplayed(ads);
+  }
+
+  @Test(
+      groups = "AbcdProductPriorityOasis"
```

Comment:
Add here specific group only for this method and try to use this group for class